dist/README.UPDATERS

```diff
@@ -1,3 +1,48 @@
+For Updaters to OBS 2.7 from OBS 2.6
+====================================
+
+WARNING: Please be aware, that this section is work in progress
+
+  1. Remove the OBS 2.6 Repository
+
+        zypper rs OBS:Server:2.6
+
+  2. Add the OBS 2.7 Repository and update repository cache
+
+        zypper ar http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_13.2/OBS:Server:2.7.repo
```

Comment:
better replace the openSUSE_13.2 part with somethink like "$YOUR_DISTRIBUTION" or people will add it to their installations even when they don't have a 13.2 due to copy&paste